userspace/libscap/CMakeLists.txt

```diff
@@ -100,8 +105,45 @@ endif()
 
 include_directories(${CMAKE_CURRENT_SOURCE_DIR})
 
+if(CMAKE_SYSTEM_NAME MATCHES "Linux")
```

Comment:
Can you share why these lines were moved above?

Reply:
I initially assumed that we might need to append the various library sources to ${targetfiles} instead of using object libraries, but the object libraries are working fine so far. I moved them back.